langchain4j-azure-open-ai/src/test/java/dev/langchain4j/model/azure/AzureOpenAiStreamingChatModelIT.java

```diff
@@ -182,8 +182,11 @@ void should_call_function_with_argument(String deploymentName, String gptVersion
         ToolSpecification toolSpecification = ToolSpecification.builder()
                 .name(toolName)
                 .description("returns a sum of two numbers")
-                .addParameter("first", INTEGER)
-                .addParameter("second", INTEGER)
+                .parameters(JsonObjectSchema.builder()
```

Comment:
nit: let's keep all tools with the old API (explained the reason in other comment)

Reply:
Change has been rolled back